optuna/storages/_base.py

```diff
@@ -540,7 +549,9 @@ def get_all_trials(self, study_id: int, deepcopy: bool = True) -> List[FrozenTri
         """
         raise NotImplementedError
 
-    def get_n_trials(self, study_id: int, state: Optional[TrialState] = None) -> int:
+    def get_n_trials(
+        self, study_id: int, state: Optional[Union[Tuple[TrialState, ...], TrialState]] = None
+    ) -> int:
         """Count the number of trials in a study.
```

Comment:
I think it is better to change the explanation of the `state` argument to match its type annotation `Optional[Union[Tuple[TrialState, ...], TrialState]]`.

Reply:
You're right, I updated the docs. Thanks!